ui/src/components/Bouts/BoutPage.tsx

```diff
@@ -0,0 +1,697 @@
+                  <TableRow>
+                    <TableCell>ID</TableCell>
+                    <TableCell>Category</TableCell>
+                    <TableCell>Description</TableCell>
+                    <TableCell align="right">Timestamp</TableCell>
+                    <TableCell align="right">Candidate</TableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {detections.map((det, index) => (
+                    <TableRow key={index}>
+                      <TableCell>
+                        <Typography variant="caption">{det.id}</Typography>
+                      </TableCell>
+                      <TableCell>
+                        <Chip label={det.category} />
+                      </TableCell>
+                      <TableCell>{det.description}</TableCell>
+                      <TableCell align="right" title={det.timestamp.toString()}>
+                        {formatTimestamp(det.timestamp)}
+                      </TableCell>
+                      <TableCell align="right">
+                        {det?.candidate?.id && (
+                          <IconButton
+                            href={`/reports/${det?.candidate?.id}`}
+                            target="_blank"
+                            size="small"
+                            sx={{ transform: "scale(0.8)" }}
+                          >
+                            <Launch />
+                          </IconButton>
+                        )}
+                      </TableCell>
+                    </TableRow>
+                  ))}
+
+                  {detections.length < 1 && (
+                    <TableRow>
+                      <TableCell colSpan={5}>
+                        <Typography textAlign="center">
+                          No detections submitted from{" "}
+                          {format(minDetectionsTime, "h:mm a")} to{" "}
+                          {format(maxDetectionsTime, "h:mm a")}
+                        </Typography>
+                      </TableCell>
+                    </TableRow>
+                  )}
+                </TableBody>
+              </Table>
+            </Box>
+          </TabPanel>
+        </Box>
+      </Box>
+    </>
+  );
+}
+
+function CategoryIcon({
+  audioCategory,
+  size,
+}: {
+  audioCategory: AudioCategory;
+  size?: number;
+}) {
+  size = size ?? 15;
+  if (audioCategory === "BIOPHONY")
+    return (
+      <Image
+        src={whaleFlukeIconImage.src}
+        width={size}
+        height={size}
+        alt="Whale fluke icon"
+      />
+    );
+  if (audioCategory === "ANTHROPHONY")
+    return (
+      <Image
+        src={vesselIconImage.src}
+        width={size}
+        height={size}
+        alt="Vessel icon"
+      />
+    );
+  if (audioCategory === "GEOPHONY")
+    return (
+      <Image
+        src={wavesIconImage.src}
+        width={size}
+        height={size}
+        alt="Waves icon"
+      />
+    );
+}
```

Comment:
untested suggestion, what do you think?

```suggestion
function CategoryIcon({
  audioCategory,
  size = 15,
}: {
  audioCategory: AudioCategory;
  size?: number;
}) {
  const iconConfig = {
    BIOPHONY: {
      src: whaleFlukeIconImage.src,
      alt: "Whale fluke icon",
    },
    ANTHROPHONY: {
      src: vesselIconImage.src,
      alt: "Vessel icon",
    },
    GEOPHONY: {
      src: wavesIconImage.src,
      alt: "Waves icon",
    },
  }[audioCategory];

  return iconConfig && (
    <Image
      src={iconConfig.src}
      width={size}
      height={size}
      alt={iconConfig.alt}
    />
  );
}
```

Reply:
Better for sure